src/client/PostClient.php

```diff
@@ -0,0 +1,42 @@
+<?php
+  require_once(dirname(__DIR__, 1)."/class/PostPayload.php");
+  require_once(dirname(__DIR__, 1)."/model/Post.php");
+
+  class PostClient {
+    public PDO $pdo;
+
+    public function __construct(PDO $pdo)
+    {
+      $this->pdo = $pdo;
+    }
```

Comment:
```suggestion
    public function __construct(
      public ?PDO $pdo = null) {
        if (is_null($pdo)) $this->pdo = PgConnect::getClient();
    }
```

とすると、単体テストが書きやすいかなと思います。